### DISCUSSION TITLE
Welcome to velo-filesystem Discussions!

### DISCUSSION THREAD

**Discussion by shoonia** [shoonia/velo-filesystem#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yalin** [comment]
> ##  Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together muscle.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hi,
Is it easy to add the feature "Download without page ids"?
I mean, for example, a page has a name "Home.2j8pq.js" and I want to download it as "Home.js" 
Ofc I can do it manually, but the idea of this project is to do things automatically, so I thought that would be so good.
And it can be optional, like settings of the extension and whoever chooses it, it downloads page names without ids.

**Reply by shoonia** [reply]
I guess we can add a checkbox for "Download without page". The main problem is that your pages may have the same duplicated name.

So, we need to check the following:
- All pages have unique names
- If a few pages have the same name, show the message "extract pages with ID, or rename one of the pages"

**Reply by shoonia** [reply]
In v0.3.0 has added checkbox "Include page ID" `true` by default. You are able to off it. If you have a few pages with the same name then you should use page ID or rename all duplicates.

**Comment by yalin** [comment]
I have another question;

I am using this extension because Wix does not have Github integration, so I am trying to keep my changes manually.
And I am using Linux. So, when I download the files, git recognizes different line endings even though there are no changes in the files. 
The ones I download through extension are; ```DOS/Windows (CR-LF)``` and the files in my local file system; ```UNIX (LF)```. Is there a way to avoid this?

**Reply by shoonia** [reply]
#38

**Reply by shoonia** [reply]
Fix in v0.10.0
